kubernetes/sig-release/v1.26/release.md

```diff
@@ -1,4 +1,6 @@
 12 月 8 号 Kubernetes 正式发布了主题为 `Electrifying` 的 v1.26。
+> Kubernetes v1.26 被命名为 “Electrifying”，中文直译就是 “电气化”，目的是提高人们对 Kubernetes 编排着消耗大量能源的巨大系统的认识。同时 “Electrifying” 也代表了发布团队使用的自动化在不断增加，并且 Kubernetes 社区也正在逐步电气化。
```

Comment:
Kubernetes 1.26 is named Electrifying to raise awareness that Kubernetes orchestrates huge systems that consume a lot of energy and this is simply a problem. Electrifying also represents the increased automation we use in the release team and the fact that this community is simply electrifying.